docs/integrate/etl/influxdb.md

```diff
@@ -0,0 +1,191 @@
+(influxdb-import)=
+# Import data from InfluxDB
+
+In this quick tutorial we use [influxio](https://github.com/daq-tools/influxio)
+to import data from InfluxDB into CrateDB. It can be used both as a standalone
+program, and as a library.
+
+:::{note}
+Please note that influxio is alpha-quality software, and a work in progress.
+Contributions of all kinds are very welcome, in order to make it more solid. 
+Breaking changes should be expected until a 1.0 release, so version pinning is
+recommended, especially when you use it as a library.
+:::
+
+(setup-influxio)=
+## Setup
+
+`influxio` can be installed with pip:
+
+:::{code} console
+pip install influxio
+:::
+
+You will also need the [cratedb-toolkit](https://github.com/crate-workbench/cratedb-toolkit/tree/main/cratedb_toolkit/io#installation)
+to load the data into your CrateDB instance. It can be installed with:
+
+:::{code} console
+pip install --upgrade 'cratedb-toolkit[all]'
+:::
+
```

Comment:
It may be sensible to add a "start services" section here, which conveys how to run both services using Docker. We can't present it for each and every platform, so Docker just yields the best platform coverage.